src/components/Popup/Popup.tsx

```diff
@@ -176,6 +176,10 @@ export default class Popup extends AutoControlledComponent<ReactProps<PopupProps
       e.stopPropagation()
     },
     close: e => this.close(e),
+    toggle: e => {
+      e.preventDefault()
```

Comment:
what is this added for?

Reply:
this is when space key trigger open popup...